### PR TITLE
return namespace assigned to --namespace

### PR DESCRIPTION
**Pull request by ashleyschuett**
**What this PR does / why we need it**:
If your local kubeconfig already has a namespace set the current logic uses that namespace instead of the namepace supplied to the `--namespace` flag during install. 

**Special notes for your reviewer**:

**If applicable**:
- [ ] this PR contains documentation
- [ ] this PR contains unit tests
- [ ] this PR has been tested for backwards compatibility

### CODE REVIEW

**Review by bacongobbler — APPROVED** [review]

**Comment by bacongobbler** [comment]
Coming back from a 3-week vacation so I'll hold off on merging until another maintainer checks this out. Need to take some time to get comfortable with the codebase again :)

**Comment by ashleyschuett** [comment]
Thank you for the update, let me know if I need to update anything!

**Review by thomastaylor312 — APPROVED** [review]
Thank you for catching this! LGTM. You may merge when ready @bacongobbler

**Pull request merged**
